HISTORY.md

```diff
@@ -170,8 +176,8 @@ This incorporates all changes after 1.3.3 up to 1.3.5.
   * Fix OPTIONS responses to include the HEAD method properly
   * Remove support for leading colon in `router.param(name, fn)`
   * Use `array-flatten` for flattening arrays
-  * deps: debug@~2.1.1
-  * deps: methods@~1.1.1
+  * deps: `debug@~2.1.1`
+  * deps: `methods@~1.1.1`
```

Comment:
Can all these extra unrelated changes be cleaned up or do you want me to review them?

Reply:
Don't think there's much "to review". I don't care about them. I'll revert them.

Reply:
Ok. Otherwise afaik this pr is good to go. I will merge it asap.

Reply:
Done. `HISTORY.md` should be clean.